evm-contracts/src/v0.6/dev/SimpleAccessControl.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity ^0.6.0;
+
+import "../Owned.sol";
+import "./AccessController.sol";
+
+/**
+ * @title SimpleAccessControl
+ * @notice Allows the owner to set access for addresses
+ */
+contract SimpleAccessControl is AccessController, Owned {
+
+  bool public checkEnabled;
+  mapping(address => bool) internal accessList;
+
+  event AddedAccess(address user);
+  event RemovedAccess(address user);
+  event CheckAccessEnabled();
+  event CheckAccessDisabled();
+
+  constructor()
+    public
+  {
+    checkEnabled = true;
+  }
+
+  /**
+   * @notice Returns the access of an address
+   * @param _user The address to query
+   */
+  function hasAccess(
```

Comment:
This function should also take into account checkEnabled.
Otherwise, there's no way to turn off access control for an EACAggregatorProxy without changing its accessController